File: samples/java/src/main/java/co/windly/limbosample/presentation/main/settings/SettingsModule.java
package co.windly.limbosample.presentation.main.settings;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class SettingsModule {

  //region Contribution

  @ContributesAndroidInjector
  abstract SettingsFragment contibuteAndroidInjector();

  //endregion
}

File: samples/kotlin/src/main/java/co/windly/limbosample/presentation/main/settings/SettingsModule.kt
package co.windly.limbosample.presentation.main.settings

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class SettingsModule {

  //region Contribution

  @ContributesAndroidInjector
  abstract fun contributeAndroidInjector(): SettingsFragment

  //endregion
}
